files/image_config/rsyslog/rsyslog.conf.j2

```diff
@@ -72,3 +68,26 @@ $RepeatedMsgReduction on
 ###############
 #### RULES ####
 ###############
+
+#
+# Remote syslog logging
+#
+
+# The omfwd plug-in provides the core functionality of traditional message forwarding via UDP and plain TCP.
+# It is a built-in module that does not need to be loaded.
+
+{% if SYSLOG_SERVER is defined %}
```

Comment:
Could you add what the config looks like in the description?
For example, the current `rsyslog.conf` looks like the below:
```
admin@vlab-01:~$ cat /etc/rsyslog.conf  | grep "^*.*"
*.* @[10.0.0.5]:514;SONiCFileFormat
*.* @[10.0.0.6]:514;SONiCFileFormat

```

Reply:
@wen587 done